google/cloud/aiplatform/explain/lit.py

```diff
@@ -0,0 +1,256 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+
+from typing import Dict, List, OrderedDict, Tuple, Union
```

Comment:
Hey, sorry that this [comment](https://github.com/googleapis/python-aiplatform/pull/874#discussion_r764111787) wasn't clear. I meant that the approach to encode the entire `OrderedDict` type as a string literal, ie: forward reference, was the right approach because `typing.OrderedDict` isn't in Python 3.6.

`OrderedDict` should be removed here and these definitions:

```Python
OrderedDict[str, lit_types.LitType]
```

should be as you had it:
```Python
"OrderedDict[str, lit_types.LitType]"
```